### ISSUE TITLE
Fractional positioning issue with rect()

### ISSUE THREAD

**Issue by pamelafox** [Khan/live-editor#429]
Reported on ZenDesk from Peter.

See example program and explanation here:
https://www.khanacademy.org/computer-programming/fractional-positioning-issue-with-rect/5953285063311360

**Comment by kevinbarabash** [comment]
I believe this should be super easy to fix, although it will require changes to processing-js.

**Comment by kevinbarabash** [comment]
Some of the commands round coordinates.  Who looks at this bug should also look at the other drawing commands and make sure that all of them draw stuff using the original numbers being passed in instead of rounding them.

**Comment by Gigabyte-Giant** [comment]
@kevinbarabash The fix for this is indeed very simple, however, they'll need to take place in [Khan/processing-js](https://github.com/Khan/processing-js), correct?

**Comment by kevinbarabash** [comment]
@Gigabyte-Giant that's right.  We should eventually try to push the changes upstream to https://github.com/processing-js/processing-js/ too.

**Comment by Gigabyte-Giant** [comment]
Sounds good... I've added this to my to-do list (http://take.ms/7FFc9); I'm going to try and fix one of the AST related issues, and then I'll take a closer look at this one.

**Comment by kevinbarabash** [comment]
Cool!
